### PR TITLE
Make output less verbose during development

### PR DESCRIPTION
**Pull request by cdupuis**
This PR changes the local default log level to info and changes a lot of internal log statements to go out on `debug` instead of `info`. All the usual information is still available from the log file.

### CODE REVIEW

File: lib/operations/common/repoUtils.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export async function doWithAllRepos<R, P>(ctx: HandlerContext,
     const promises = ids.map(id =>
         () => repoLoader(id)
             .catch(err => {
-                logger.warn("Unable to load repo %s/%s: %s", id.owner, id.repo, err);
+                logger.debug("Unable to load repo %s/%s: %s", id.owner, id.repo, err);
```
Comment (ddgenome) [inline lib/operations/common/repoUtils.ts]: I've noticed a few places where failures that were logged as warnings are now debug. I presume you have your reasons…

File: lib/project/git/GitCommandGitProject.ts [hunk 2]
```diff
@@ -372,15 +372,15 @@ async function cloneInto(
     await promiseRetry(retryOptions, (retry, count) => {
         return execPromise("git", cloneArgs)
             .catch(err => {
-                logger.warn(`Clone of ${id.owner}/${id.repo} attempt ${count} failed: ` + err.message);
+                logger.debug(`Clone of ${id.owner}/${id.repo} attempt ${count} failed: ` + err.message);
```
Comment (ddgenome) [inline lib/project/git/GitCommandGitProject.ts]: This one seems like a legitimate change.

File: lib/project/git/GitCommandGitProject.ts [hunk 3]
```diff
@@ -372,15 +372,15 @@ async function cloneInto(
...
                 retry(err);
             });
     });
     try {
         await execPromise("git", ["checkout", checkoutRef, "--"], { cwd: repoDir });
     } catch (err) {
         // When the head moved on and we only cloned with depth; we might have to do a full clone to get to the commit we want
-        logger.warn(`Ref ${checkoutRef} not in cloned history. Attempting full clone`);
+        logger.debug(`Ref ${checkoutRef} not in cloned history. Attempting full clone`);
```
Comment (ddgenome) [inline lib/project/git/GitCommandGitProject.ts]: Legit

**Comment by cdupuis** [comment]
Pull request auto merged by Atomist.

* 1 approved review by @ddgenome
* 2 successful checks

[atomist:generated] [auto-merge:on-approve]

**Pull request merged**